resources/views/entities/components/_post.blade.php

```diff
@@ -98,3 +154,25 @@
         </div>
     </div>
 </div>
+
+@section('styles')
+    @parent
+    @vite('resources/sass/abilities.scss')
```

Comment:
These should be added in the abilities subpage, otherwise we are loading a ton of extra css/js even when we don't need any of it or when there is no special layout